nuxeo-coldstorage/src/test/java/org/nuxeo/coldstorage/service/TestDummyColdStorageService.java

```diff
@@ -163,7 +163,7 @@ public void shouldFireDedicatedDownloadEvent() {
 
             // and assert it is cancelled and a cold storage download event is fired instead
             assertEquals(2, listener.streamCapturedEvents().count());
-            assertTrue(listener.findFirstCapturedEvent(DownloadService.EVENT_NAME).get().isCanceled());
+            listener.findFirstCapturedEvent(DownloadService.EVENT_NAME).ifPresent(e -> assertTrue((e.isCanceled())));
```

Comment:
This change doesn't have the same semantic, now if the event is not fired the test will succeed whereas it would have failed previously.